tests/support/globalSetup.ts

```diff
@@ -1,31 +1,105 @@
-  } catch (err) {
-    console.error(err);
-    throw new Error(notRunningMessage);
-  }
+  console.log("Setting up global test environment");
+  console.log("Resetting environment");
+  const tmp = Path.resolve(dir, "..", "./tmp");
+  const projectName = "source-browsing";
+  const sourceBrowsingDir = Path.join(tmp, "repos", projectName);
+  await Fs.rm(Path.join(tmp, "peers"), { recursive: true, force: true });
+  await Fs.rm(Path.join(tmp, "repos"), { recursive: true, force: true });
+  await Fs.mkdir(Path.join(sourceBrowsingDir), { recursive: true });
+  await Process.spawn("tar", [
+    "-xf",
+    Path.resolve(dir, "..", `./fixtures/repos/${projectName}.tar.bz2`),
+    "-C",
+    sourceBrowsingDir,
+  ]);
+
+  console.log("Create project source-browsing");
+  const peerManager = await createPeerManager({
+    dataDir: Path.resolve(dir, "..", "./tmp/peers"),
+    outputLog: FsNode.createWriteStream(Path.resolve(dir, "..", "./tmp/log")),
+  });
+  const palm = await peerManager.startPeer({ name: "palm" });
+  await palm.startHttpd();
+  const alice = await peerManager.startPeer({
+    name: "alice",
+    gitOptions: gitOptions["alice"],
+  });
+  const bob = await peerManager.startPeer({
+    name: "bob",
+    gitOptions: gitOptions["bob"],
+  });
+  await palm.startNode({ trackingPolicy: "track", trackingScope: "all" });
+  await alice.startNode({ connect: palm });
+  await bob.startNode({ connect: palm });
+  await sleep(1000);
+  await alice.git(["clone", sourceBrowsingDir], { cwd: alice.checkoutPath });
+  await alice.git(["checkout", "main"]);
+  await alice.rad([
+    "init",
+    "--name",
+    projectName,
+    "--default-branch",
+    "main",
+    "--description",
+    "Git repository for source browsing tests",
+    "--announce",
+  ]);
+
+  await alice.git(["checkout", "feature/branch"]);
+  await alice.git(["push", "rad"]);
+  await alice.git(["checkout", "orphaned-branch"]);
+  await alice.git(["push", "rad"]);
+
+  const { stdout: rid } = await alice.rad(["inspect"]);
+
+  await alice.rad(["track", bob.nodeId]);
+  await sleep(2000);
+  await bob.rad(["clone", rid], { cwd: bob.checkoutPath });
+  await sleep(2000);
+
+  console.log("Modify project source-browsing");
+  await Fs.writeFile(
+    Path.join(bob.checkoutPath, "source-browsing", "README.md"),
+    "Updated readme",
+  );
+  await bob.git(["add", "README.md"]);
+  await bob.git([
+    "commit",
+    "--message",
+    "Update readme",
+    "--date",
+    "Mon Dec 21 14:00 2022 +0100",
+  ]);
+  await bob.git(["push", "rad"]);
+}
 
-  if (nodeId !== seedRemote) {
-    const wrongSeedMessage = `The server on port ${seedPort} doesn't have the right fixtures.\n`;
-    throw new Error(wrongSeedMessage + notRunningMessage);
+// Assert that the `rad` CLI is installed and has the correct version.
+async function assertRadInstalled(): Promise<void> {
+  const result = await execa("rad", ["--version"]);
+  const versionConstraint = "rad 0.8.0 (a6a3290)";
```

Comment:
As discussed, I think it would be good to have the heartwood version in a single place. Like for example at the top-lovel of the repo in `heartwood-version` and then have the SHA in there and source it in all places where we need the version.